adapter/redisQuota/config/config.proto

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package adapter.redisQuota.config;
+
+import "google/protobuf/duration.proto";
+
+option go_package = "config";
+
+message Params {
+    // Minimum number of seconds that deduplication is possible for a given operation.
+    google.protobuf.Duration min_deduplication_duration = 1;
+
+    // Redis network address
```

Comment:
nit: you might consider dropping `redis_` from all of the fields, as this is a config proto only exposed to the redisQuota package -- and it is clear that Url, SocketType, and PoolSize apply to redis itself.